.github/workflows/updates-e2e.yml

```
@@ -61,9 +61,12 @@ jobs:
       - name: Init new expo app
         working-directory: ../
         run: expo-cli init updates-e2e --yes
-      - name: Add local expo-updates and dependencies
+      - name: Add yarn resolutions for local dependencies
         working-directory: ../updates-e2e
-        run: yarn add file:../expo/packages/expo-updates file:../expo/packages/expo file:../expo/packages/expo-eas-client-id file:../expo/packages/expo-modules-core file:../expo/packages/expo-json-utils file:../expo/packages/expo-manifests file:../expo/packages/expo-structured-headers file:../expo/packages/expo-updates-interface
+        run: 'node -e "console.log(JSON.stringify({...require(''./package.json''), resolutions: {''expo-eas-client-id'': ''file:../expo/packages/expo-eas-client-id'', ''expo-modules-core'': ''file:../expo/packages/expo-modules-core'', ''expo-json-utils'': ''file:../expo/packages/expo-json-utils'', ''expo-manifests'': ''file:../expo/packages/expo-manifests'', ''expo-structured-headers'': ''file:../expo/packages/expo-structured-headers'', ''expo-updates-interface'': ''file:../expo/packages/expo-updates-interface''}}, null, 2))" > new-package.json && mv new-package.json package.json'
```

The macos-11 runner [comes with jq installed](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#utilities).

```suggestion
        run: |
          # A jq filter to add a "resolutions" field
          JQ_FILTER=$(cat << EOF
            . + {
              resolutions: {
                "expo-eas-client-id": "file:../expo/packages/expo-eas-client-id"
              }
            }
          EOF
          )

          jq "$JQ_FILTER" < package.json > new-package.json
          mv new-package.json package.json
```